README.md

```diff
@@ -126,3 +126,72 @@ Now, for any new html page, we need to do the following:
 {% endblock %}
 
 ```
+
+## pre deployment to fly.io
+
+
+- Setting Up environmental variables
+
+  - Add `python-dotenv>=0.21.0` to requirements.txt
+  - Run `pip install -r requirements.txt`
```

Comment:
Do you need this for fly or only for local development? If local development that should be a separate section of the README

Reply:
i need it for fly to put secret key, csrf_trusted_origins, database_url and development mode.

Reply:
Great then please make commands that can be copy and pasted into the terminal.

```
  - Run `pip install -r requirements.txt`
```

is not a valid bash command.